app/src/main/res/values/strings.xml

```diff
@@ -177,26 +180,26 @@
     <string name="restore_next">Next</string>
     <string name="restore_restoring">Restoring backup</string>
     <string name="restore_restoring_error_title">Unable to restore some apps</string>
-    <string name="restore_restoring_error_message">You can re-install these apps manually and "Automatic restore" will attempt to restore their data (when enabled).</string>
-    <string name="restore_magic_package">System package manager</string>
-    <string name="restore_finished_success">Restore complete</string>
-    <string name="restore_finished_error">An error occurred while restoring the backup.</string>
+    <string name="restore_restoring_error_message">You can re-install these apps manually and "Auto-restore" will attempt to restore their data (when on).</string>
+    <string name="restore_magic_package">System package-manager</string>
+    <string name="restore_finished_success">Restoration complete</string>
+    <string name="restore_finished_error">Could not restore the backup.</string>
     <string name="restore_finished_button">Finish</string>
 
     <string name="restore_storage_skip">Skip restoring files</string>
     <string name="restore_storage_choose_snapshot">Choose a storage backup to restore (experimental)</string>
```

Comment:
What exactly is a storage backup?
Full partition backup?

Reply:
No, you get to choose which part of the (Internal) storage you want to backup.